src/common/Network/PacketDef/Zone/ServerZoneDef.h

```diff
@@ -283,6 +283,100 @@ namespace Sapphire::Network::Packets::Server
     char unk2[0x4]; // This has probs something to do with the support desk (inquiry id?)
   };
 
+  struct FFFXIVIpcMarketBoardItemListingCount : FFXIVIpcBasePacket< MarketBoardItemListingCount >
+  {
+    uint32_t itemCatalogId;
+    uint32_t unknown1; // does some shit if nonzero
+    uint16_t requestId;
+    uint16_t quantity; // high/low u8s read separately?
+    uint32_t unknown3;
+  };
+
+  struct FFXIVIpcMarketBoardItemListing : FFXIVIpcBasePacket< MarketBoardItemListing >
+  {
+    struct itemListing // 152 bytes each
+    {
+      uint32_t unknown; // Changes if multiple packets are sent for a single item
+      uint8_t padding;
+      uint16_t unknown1;
+      uint8_t padding1;
+      struct unknown2
+      {
+        uint32_t unknown3;
+        uint16_t unknown4;
+        uint16_t unknown5;
+      } unknown6[3];
+      uint32_t pricePerUnit;
+      uint32_t unknown7;
+      uint32_t itemQuantity;
+      uint32_t itemID;
+      uint16_t unknown8;
+      uint16_t unknown9;
+      uint32_t unknown10;
+      uint16_t unknown11; // Always 30000
+      uint16_t padding3;
+      uint16_t materiaValue[5]; // Materia ID = (materiaValue & 0xFF0) >> 4
+      uint32_t padding4;
+      char retainerName[64];
+      bool hq;
+      uint8_t materiaCount;
+      uint8_t padding5;
+      uint8_t retainerCity;
+      /**
+       * 0x01 Limsa Lominsa
+       * 0x02 Gridania
+       * 0x03 Ul'dah
+       * 0x04 Ishgard
+       * 0x07 Kugane
+       * 0x0A Crystarium
+       */
+      uint64_t padding6;
+    } listing[10]; // Multiple packets are sent if there are more than 10 search results.
+    uint32_t unknown12;
+    char padding7[16];
+    uint8_t unknown13;
+    uint16_t padding8;
+    uint8_t unknown14;
+    uint64_t padding9;
```

Comment:
not sure what this is, but there would be a u64 for the content id of a character so you can request the signature on an item, might be this, might be elsewhere in the packet

Reply:
Just double-checked, that's always empty. padding6 is also always empty. I don't know what content IDs are supposed to look like, but parsing unknown8 through unknown10 as a uint64 gives numbers like `8444249366530776407` back, so maybe that's it?

Reply:
it'll be requested via this actrl https://github.com/SapphireServer/Sapphire/blob/develop/src/common/Network/CommonActorControl.h#L322 and then you'll get this packet back https://github.com/SapphireServer/Sapphire/blob/develop/src/common/Network/PacketDef/Zone/ServerZoneDef.h#L1124-L1128

Reply:
I don't seem to be getting back any packets with item creator names, which is strange, but there are definitely no outbound packets with that uint64, which rules that range out if nothing else